.github/workflows/maya-hydra-preflight-launcher.yml

```diff
@@ -11,7 +11,7 @@ jobs:
 
 #  clang_format_linter:
 #      timeout-minutes: 30
-#      runs-on: ubuntu-latest
+#      runs-on: ubuntu-22.04
```

Comment:
In MayaUsd our clang-format step started failing all of a sudden since the unbuntu-latest (24.04) was updated to Python 3.12.